variables.mk

```diff
@@ -221,6 +221,10 @@ sim_files              ?= $(build_dir)/sim_files.f
 # single file that contains all files needed for VCS or Verilator simulation (unique and without .h's)
 sim_common_files       ?= $(build_dir)/sim_files.common.f
 
+SFC_LEVEL ?= $(build_dir)/.sfc_level
+FINAL_EXTRA_FIRRTL_OPTIONS ?= $(build_dir)/.final_extra_firrtl_options
```

Comment:
Nit : Can we use variable names like `EXTRA_FIRRTL_OPTIONS_BASE` & `EXTRA_FIRRTL_OPTIONS` instead of `EXTRA_FIRRTL_OPTIONS` & `FINAL_EXTRA_FIRRTL_OPTIONS` to match the formatting with `MFC_BASE_LOWERING_OPTIONS` and `MFC_LOWERING_OPTIONS`?